agent/src/main/java/kanela/agent/util/KanelaInformationProvider.java

```diff
@@ -32,6 +32,7 @@ private static String moduleJson(KanelaConfiguration.ModuleConfiguration module)
                 .value("description", module.getName())
                 .value("enabled", module.isEnabled())
                 .value("active", registryListener.isModuleActive(module.getKey()))
+                .value("version", registryListener.modeuleVersion(module.getKey()))
```

Comment:
`modeuleVersion` typo? ;)

Reply:
@damdev also I've created a class with some utils for MANIFESTS `https://github.com/kamon-io/kanela/blob/master/agent/src/main/java/kanela/agent/util/Manifests.java`

Reply:
Done! I move the extractVersion method to the Manifests class